app-frontend/src/app/components/filterPane/filterPane.controller.js

```diff
@@ -26,49 +28,73 @@ export default class FilterPaneController {
         }
     }
 
-    close() {
-        this.opened = false;
-    }
-
-    onYearFiltersChange(id, minModel, maxModel) {
-        if (minModel && maxModel) {
-            if (minModel === this.yearRange.min) {
-                delete this.filters.minAcquisitionDatetime;
-            } else {
-                this.filters.minAcquisitionDatetime =
-                    (new Date(minModel, 0, 1))
-                    .toISOString();
+    initDatefilter() {
+        this.datefilter = {
+            start: moment().subtract(100, 'years'),
+            end: moment()
+        };
+        this.dateranges = [
+            {
+                name: 'Today',
+                start: moment(),
+                end: moment()
+            },
+            {
+                name: 'Last Month',
```

Comment:
I think this should be 'The last month' to indicate its the last 30 days and not just the entire month before this one.  This would apply to the year one as well...'The last year' vs 'Last Year'. It's a bit weird so I googled it to see: http://english.stackexchange.com/questions/62722/last-year-vs-the-last-whole-of-the-last-year-vs-whole-the-last-year